woocommerce/woocommerce-android
# [TTPA CA/UK] Add the TTP pin explanation screen

Closes: #9880


### Description


The PR adds TTP about the screen that explains how to use the feature and its restrictions (PIN). PIN sections are visible only for the countries where restrictions in place (CA and UK)

### Testing instructions

Check the screen on the stores in CA/UK/US. Light and Dark mode
* More -> Payments ->  About Tap To Pay. Notice the screen. Notice that important sections is visible in the CA and UK and the test is adapted for the country and PIN limit
* Click on "learn more about card readers"

### Images/gif


https://github.com/woocommerce/woocommerce-android/assets/4923871/877575c5-8083-4072-8255-b2e0e5f27c3f
